fr/integrations/plugin-packs/procedures/applications-nginx-plus-restapi.md

```diff
@@ -0,0 +1,159 @@
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+### Host
+
+Ce Plugin-Pack est conçu de manière à avoir dans Centreon un hôte par environnement Nginx Plus
+Lorsque vous ajoutez un hôte à Centreon, appliquez-lui le modèle *App-Nginx-Plus-Restapi-custom*. Une fois celui-ci configuré, certaines macros doivent être renseignées:
+
+| Mandatory | Name        | Description                                      |
+| :-------- | :---------- | :----------------------------------------------- |
+| X         | APIPORT     | Port used (Default: 443)                         |
+| X         | APIPROTO    | Specify https if needed (Default: 'https')       |
+| X         | APIUSERNAME | Nginx basic username                             |
+| X         | APIPASSWORD | Nginx basic password.                            |
+| X         | APIPATH     | Specify api path (Default: '/api/6')             |
+
+
+## FAQ
+
+### Comment tester un contrôle en ligne de commandes et que signifient les options principales ?
+
+Une fois le Plugin installé, vous pouvez tester celui-ci directement en ligne de commandes depuis votre collecteur Centreon avec l'utilisateur *centreon-engine*:
+
+```bash
+/usr/lib/centreon/plugins/centreon_nginx_plus_restapi.pl \
+    --plugin=apps::nginx::nginxplus::restapi::plugin \
+	--mode=connections \
+	--port='443' \
+	--proto='https' \
+	--api-username='myapiuser' \
+	--api-password='myapipassword' \
+	--api-path='/api/6' \
+	--filter-counters='mycountersfilter' \
+	--warning-active='60' \
+	--critical-active='80'
+	--warning-idle='8' \
+	--critical-idle='10' \
+    --warning-accepted='50' 
+	--critical-accepted='65' \
+	--warning-dropped='3' \
+    --critical-dropped='5' \
+	--verbose
+	
+
+OK: Active : 5, Idle : 0, Accepted : 5, Dropped : 0|
+'connections.active.count'=5;;60;80; 'connections.idle.count'=1;;8;10; 'connections.accepted.count'=5;;50;65; 'connections.dropped.count'=0;;3;5;
+```
+
+La commande ci-dessus contrôle les connexions de Nginx Plus (```--mode=connections```).
+Cette commande déclenchera une alarme WARNING si les connexions activent dépasse 60 (```--warning-active='60'```) 
+et une alarme CRITICAL s'il dépasse 80 (```--critical-active='80'```).
+
+
+Toutes les options et leur utilisation peuvent être consultées avec le paramètre ```--help``` ajouté à la commande:
+
+```/usr/lib/centreon/plugins/centreon_nginx_plus_restapi.pl --plugin=apps::nginx::nginxplus::restapi::plugin --mode=connections --help```
+
+
+### Comment puis-je supprimer les perfdatas *count* dans le cas où je ne souhaite vérifier qu'une seule application ?
+
+Le Plugin permet de filtrer sur un ou plusieurs éléments mais permet également de récupérer l'ensemble des éléments si aucun filtre n'est spécifié.
+De ce fait, des perfdatas "globales" sur les statistiques des objets sont ajoutées par défaut. Il est possible de supprimer ces données de performance en appliquant le filtre suivant: ```--filter-perfdata='^$'```.
```

Comment:
Add documentation about the most commons error when using restapi plugin:
https://docs.centreon.com/current/fr/integrations/plugin-packs/procedures/network-cisco-meraki-restapi.html#jobtiens-le-message-derreur-suivant